### DISCUSSION TITLE
Not able to access

### DISCUSSION THREAD

**Discussion by CobraDesign47** [Donkie/Spoolman#354] [Q&A] [answered]
Hello! I just installed Spoolman without Docker. I selected to not install it as a service, as I thought that it would interfere with KlipperScreen. I'm having trouble accessing it from my laptop, and I'm getting "Spoolman Server not available" notifications in Mainsail. 
Could someone help?

**Comment by Donkie** [comment, marked as answer]
Run the install again but install it as a service. It wont interfer with your screen.

If its not a service, you will need to ensure yourself that Spoolman is running, which it isnt by default.

**Reply by CobraDesign47** [reply]
Hello! Thanks for the response! Do I have to uninstall it, or do I just install it again without uninstalling anything?

**Reply by Donkie** [reply]
Just install again, no need to uninstall

**Reply by CobraDesign47** [reply]
Thank you so much! The 3D printing community needs more cool projects like this!
